editor/editor_settings.cpp

```diff
@@ -314,7 +314,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("interface/editor/quit_confirmation", true);
 
 	_initial_set("interface/theme/preset", 0);
-	hints["interface/theme/preset"] = PropertyInfo(Variant::INT, "interface/theme/preset", PROPERTY_HINT_ENUM, "Default,Grey,Godot 2,Arc,Light,Alien,Custom", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_RESTART_IF_CHANGED);
+	hints["interface/theme/preset"] = PropertyInfo(Variant::INT, "interface/theme/preset", PROPERTY_HINT_ENUM, "Default,Grey,Godot 2,Arc,Light,Alien,Solarized (Dark),Solarized (Light),Custom", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_RESTART_IF_CHANGED);
```

Comment:
That means that all users currently using Custom will be automatically moved to Solarized (Dark), no?

Reply:
Looks like, but I can add these themes after Custom

Reply:
Yeah but it would look weird I guess. Maybe Custom should be the 2nd entry after "Default", and additional themes can go after that. It will break the theme choice for everyone this time, but it should be future proof afterwards.

Reply:
Ok

Reply:
@akien-mga Done